packages/components/src/mobile-promo-card/index.tsx

```diff
@@ -103,7 +118,11 @@ export default function MobilePromoCard( { className, isWoo }: MobilePromoCardPr
 					{ isWoo && <img src={ iconWoo } alt="Icon for the Woo mobile app" /> }
 					{ ! isWoo && <WordPressJetpackSVG /> }
 				</div>
-				<p className="promo-card__title">{ getTitle() }</p>
+				<p className="promo-card__title">
+					{ isWoo
+						? translate( 'Bring your Store stats with you using the Woo mobile app' )
```

Comment:
Previous `Woo` and `Jetpack` strings were also links. Is this change intentional?

Reply:
Yes, this was intentional in response to some design feedback.